src/handlebars/installers.handlebars

```diff
@@ -0,0 +1,169 @@
+{{> header title='Installers | ' style1='<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/styles/obsidian.min.css">' }}
+
+<main class="grey-bg">
+
+  <h1 class="large-title">Installers</h1>
+
+  <div class="align-left support">
+    <div class="margin-bottom">
+      <p>Installers are currently available for Windows and macOS JDK and JRE packages.</p>
```

Comment:
Mac OS X

Reply:
I always hesitate when I have to write it because they keep changing it! Mac OS X is for 10.7 and earlier, OS X for 10.8 - 10.11 and macOS is the latest from 10.12. I thought we ought to be referring to it by the latest, but am happy to use Mac OS X if you think we ought to. (Ref: https://en.wikipedia.org/wiki/MacOS)

Reply:
I completely missed that - yeah what you have is fine then